src/Stryker.CLI/Stryker.CLI/OptionsBuilder.cs

```diff
@@ -59,11 +65,17 @@ public class OptionsBuilder
                 {   
                     return config.GetSection(thresholdOptionsSectionKey).GetValue<T>(defaultValue.JsonKey);
                 }
+                else if (config.GetSection("files-to-exclude").Exists() &&
```

Comment:
The configuration section should not have been loaded in this method but since it has already slipped through with a previous pr i'll allow it here and create a new issue to find a better place for both files-to-include and threshold-options

Reply:
Yeah, I thought already that shouldn't be there. But due to the fact that the CLI and the stryker options file are merged it is rather difficult to split them in a nicer way. When accepted I'll search for a solution.

Reply:
I created #203 to solve this issue after your PR has been merged.

Reply:
Ok cool, I'll resolve it for now.